internal/security/grantvmgroupaccess.go

```diff
@@ -88,21 +99,24 @@ func GrantVmGroupAccess(name string) error {
 	sd := uintptr(0)
 	origDACL := uintptr(0)
 	if err := getSecurityInfo(fd, uint32(ot), uint32(si), nil, nil, &origDACL, nil, &sd); err != nil {
-		return errors.Wrapf(err, "%s GetSecurityInfo %s", gvmga, name)
+		return fmt.Errorf("%s GetSecurityInfo %s: %w", gvmga, name, err)
 	}
 	defer syscall.LocalFree((syscall.Handle)(unsafe.Pointer(sd)))
 
+	if desiredAccess == 0 {
```

Comment:
Ok last nit. IMO since we have the overload now this should be an error case. If you call `GrantWithMask` you need to provide a valid mask. This would also give a clean error to the tool rather than silently providing read to the file when no flags are specified. But your call.

Reply:
no, I think that makes sense.

Reply:
To keep the original behavior, I default to `AccessMaskRead` if no flags are provided, I updated the help accordingly. lmk if you think I should change it.